### ISSUE TITLE
Error building Hugo site from readme instructions

### ISSUE THREAD

**Issue by stephen-rowan** [C3ETH/c3eth.github.io#82]
Hi there,

Using a Ubuntu image on VirtualBox I followed the README instructions as follows -

`sudo apt-get git`

`sudo apt install nodejs`
`sudo apt install npm`

`git clone --recurse-submodules --depth 1 https://github.com/C3ETH/c3eth.github.io.git`
`cd c3eth.github.io`

`git submodule update --recursive`
`git pull --recurse-submodules`

`sudo npm install -D --save autoprefixer`
`sudo npm install -D --save postcss-cli`

All fine until
`hugo server`

returned this error

![VirtualBox_ETH_06_01_2022_15_11_15](https://user-images.githubusercontent.com/25156451/148404614-4fda5634-be37-4a13-b12b-3a9101e7af49.png)

**Comment by wolstaeb** [comment]
Appears to be some problem with an image format with featured-background-1.jpg so that indicates it is either missing or corrupt. Can you check that the image featured-background-1.jpg is present and can be viewed?

**Comment by stephen-rowan** [comment]
Hi Robert. After this I did check and featured-background-1.jpg was present (but I did not try to view it). I actually deleted the file to see if it was breaking the build and the error moved onto another jpg file (implying all image files are breaking the build ?).

This morning I spun up VirtualBox and tried it again ...
![VirtualBox_ETH_08_01_2022_08_55_24](https://user-images.githubusercontent.com/25156451/148638451-8208a381-05e8-4884-a3a0-1c03ee598d7a.png)

This time it tripped on featured-sunset.jpg. So I checked whether I can view that file as you suggested and it does look like it is corrupt ...

![VirtualBox_ETH_08_01_2022_09_03_59](https://user-images.githubusercontent.com/25156451/148638580-0ca9bffe-c6d9-4061-a6b2-7005905556c2.png)

This suggests to me that either all image files are breaking the build (both jpg or png) or the shortcode is that looks for "featured" is broken ?

I could wipe the VM and start again with another clone - to see if this reproduces if you like.

S
